kv/urm.go

```diff
@@ -343,6 +471,11 @@ func (s *Service) deleteUserResourceMapping(ctx context.Context, tx Tx, filter i
 	if err := b.Delete(key); err != nil {
 		return UnavailableURMServiceError(err)
 	}
+
+	if err := ib.Delete(ikey); err != nil {
+		return UnavailableURMServiceError(err)
```

Comment:
One thing I just realized is this will fail for authorization net yet in index. Which is a plausible scenario. Perhaps we should just log the error in this situation and continue with the delete?

Reply:
Actually, I think Delete doesn't return an error when key is not found. Just validating that now.

Reply:
Yeah we're good ignore me.